### ISSUE TITLE
How to use transductive regressor without using inductive regressor

### ISSUE THREAD

**Issue by JamesRuuu** [Zhicaiwww/Bi-VAEGAN#7]
Hi, I still have some questions about how to get Top1 accuracy of AwA2. From table 3, it tells using FN and TR can achieve the best result. But how do you guys avoid using inductive regressor when you use transductive regressor? I didn't find the way you implement in the code.  By the way, you have spelt incorrectly in cub.sh. It's 'beta' not 'beat'.

**Comment by Zhicaiwww** [comment]
Thank you for your interest in this work. I have reviewed the text as well as the [issue#6](https://github.com/Zhicaiwww/Bi-VAEGAN/issues/6) and made some corrections:

- I modified the training script in the latest commit for reproducibility . To get the results in the paper, the radius $r$ should be set to a larger value rather than the default 1.

- Regarding the transductive setting, the regressor itself remains unchanged compared to the inductive setting. The only difference lies in how we use the unlabeled unseen data to train the regressor. If you want to reproduce the results under the inductive setting, simply remove the '--transductive' flag from the scripts.
